mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -276,6 +280,7 @@ class AFTSurvivalRegression @Since("1.6.0") (@Since("1.6.0") override val uid: S
     val intercept = parameters(1)
     val scale = math.exp(parameters(0))
     val model = new AFTSurvivalRegressionModel(uid, coefficients, intercept, scale)
+    instr.logSuccess(model)
```

Comment:
minor: move the `logSuccess` call after the `copyValues`. That way if we ever do something in `logSuccess` like logging parameters of the model, the call will reflect the copied parameters.